products/deploymentmanager/api.yaml

```diff
@@ -0,0 +1,177 @@
+        kind: 'deploymentmanager#operation'
+        path: 'name'
+        full_url: 'selfLink'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error/errors'
+        message: 'message'
+    parameters:
+      # These properties are query parameters given on create/update/delete.
+      # They should be tracked and updatable.
+      - !ruby/object:Api::Type::Enum
+        name: 'createPolicy'
+        description: |
+          Set the policy to use for creating new resources. Only used on
+          create and update. Valid values are `CREATE_OR_ACQUIRE` (default) or
+          `ACQUIRE`. If set to `ACQUIRE` and resources do not already exist,
+          the deployment will fail. Note that updating this field does not
+          actually affect the deployment, just how it is updated.
+        url_param_only: true
+        default_value: :CREATE_OR_ACQUIRE
+        values:
+          - :ACQUIRE
+          - :CREATE_OR_ACQUIRE
+      - !ruby/object:Api::Type::Enum
+        name: 'deletePolicy'
+        description: |
+          Set the policy to use for deleting new resources on update/delete.
+          Valid values are `DELETE` (default) or `ABANDON`. If `DELETE`,
+          resource is deleted after removal from Deployment Manager. If
+          `ABANDON`, the resource is only removed from Deployment Manager
+          and is not actually deleted. Note that updating this field does not
+          actually change the deployment, just how it is updated.
+        url_param_only: true
+        default_value: :DELETE
+        values:
+          - :ABANDON
+          - :DELETE
+      - !ruby/object:Api::Type::Boolean
+        name: 'preview'
+        description: |
+          If set to true, a deployment is created with "shell" resources
+          that are not actually instantiated. This allows you to preview a
+          deployment. It can be updated to false to actually deploy
+          with real resources.
+        default_value: false
+        url_param_only: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          Unique name for the deployment
+        required: true
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: |
+          Optional user-provided description of deployment.
+      - !ruby/object:Api::Type::Fingerprint
+        name: 'fingerprint'
+        description: |
+          Output only. Fingerprint generated by Deployment Manager for use in
+          modification requests.
+      - !ruby/object:Api::Type::Array
+        name: 'labels'
+        description: |
+          Key-value pairs to apply to this address.
```

Comment:
What does `address` apply to here? Should this be `deployment`?

Reply:
herp